manager/volume.go

```diff
@@ -501,6 +497,26 @@ func (m *VolumeManager) Activate(volumeName string, frontend string) (v *longhor
 	return v, nil
 }
 
+func (m *VolumeManager) triggerBackupVolumeToSync(volume *longhorn.Volume) error {
+	backupVolumeName, isExist := volume.Labels[types.LonghornLabelBackupVolume]
+	if !isExist || backupVolumeName == "" {
+		logrus.Debugf("cannot find the backup volume label: %v", volume.Name)
+		return nil
+	}
```

Comment:
Shouldn't return error instead? because `Activate` method is specific for DR volume. 

nit: Also, why the label would be possible missing due to human intervention? then we probably need to improve the update mutation.

Reply:
- Got it, I will return error instead of only log the label missing. 
 - ticket for improving missing labels: https://github.com/longhorn/longhorn/issues/5433